README.md

```diff
@@ -1,4 +1,42 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql/README.md) In-progress
-2. [Core Java Apps](./core_java/README.md) In-progress
-3. [Springboot Trading REST API](./springboot/README.md) In-progress
+# David Yang . Jarvis Consulting
+
+I am a graduate from the University of Toronto with a degree in Electrical and Computer Engineering. Professionally, I have had the opportunity to take on a few months of software 
+work experience writing perl, python & tcl scripts to automate and report data in human readable tables and graphs. With automation, I was able to cut time spent on analyzing 
+data by more than half. In addition to work experience, I enjoyed topics related to big data in school - my favorites being deep learning and databases. Outside of work I enjoy 
+playing badminton and simulation video games. One game I enjoy in particular is City Skylines as I enjoy creating custom maps as well as testing other people's mods and others 
```

Comment:
Please run a spelling and grammar check using https://www.grammarly.com/ if necessary